### DISCUSSION TITLE
如何链接到非markdown的本地文件如pdf/word/xlsx？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#24] [Q&A常见问题] [answered]
我尝试
`* [afile](..\asset\分析.xlsx)`

不过会出现报错，找不到 `..\asset\分析.xlsx.md`

**Comment by Quorafind** [comment, marked as answer]
用 FILE 命令链接，例如`file:///C:/Bon/Bon.docx`

**Reply by makerjackie** [reply]
可以了，感谢 `[某个文件](file:///D://workspace/docs/jackieworld/docs/asset/obsidian.pdf)` 这样之后，在预览模式下（按Ctrl + E进入），点击，或者在编辑模式下，Ctrl+点击，就可以跳转了

**Reply by makerjackie** [reply]
不过似乎不支持相对路径
